airflow/cli/commands/kubernetes_command.py

```diff
@@ -90,7 +90,7 @@ def cleanup_pods(args):
     print('Loading Kubernetes configuration')
     kube_client = get_kube_client()
     print(f'Listing pods in namespace {namespace}')
-    continue_token = None
+    continue_token = ''
```

Comment:
Are you sure an empty string works ?

https://github.com/kubernetes-client/python/blob/v11.0.0/kubernetes/client/models/v1_list_meta.py#L56-L57

https://github.com/kubernetes-client/python/blob/v12.0.0/kubernetes/client/models/v1_list_meta.py#L61-L62

Reply:
@kaxil we don't support python client v12.

Reply:
https://github.com/apache/airflow/commit/22415e8271980bd13d88efc9598e20f766ba00d8#diff-b6af878da7897f307bf19ee1137d525e6da262bc2fa6469ce2c40f260be7db48R30 
@kaxil continue is None will raise error in this kubernetes test